### DISCUSSION TITLE
blog/2023/09/18/understanding-dirichlet-beta-intuition/

### DISCUSSION THREAD

**Discussion by giscus** [andrewheiss/ath-quarto#92] [Blog comments]
# blog/2023/09/18/understanding-dirichlet-beta-intuition/

Learn about the Dirichlet distribution and explore how it’s just a fancier version of the Beta distribution

https://www.andrewheiss.com/blog/2023/09/18/understanding-dirichlet-beta-intuition/

**Comment by Robinlovelace** [comment]
This is really great stuff Andrew, many thanks for sharing!

**Comment by kane9530** [comment]
I love and look forward to your blog posts. They are amazingly high-quality and instructive!

**Comment by jacabenga** [comment]
Very useful. Thanks.
